Extension/package.json

```diff
@@ -59,7 +59,7 @@
             "id": "verify.compiler.mac",
             "title": "Install a C++ compiler",
             "description": "The C++ extension uses the C++ compiler on your system to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
-            "when": "isMac",
+            "when": "workspacePlatform == mac",
```

Comment:
Is this in the latest insiders yet?  I just tried it out, and it didn't seem to work?  Also, I'm curious what happens if no workspace is open.  Does it default the host platform, or do we need some boolean logic to use the original flags if no workspacePlatform?

Reply:
It should be in tonight's Insiders. The commit is at https://github.com/microsoft/vscode/commit/196e79256dc9fcda1bdc93f0ac321223d8ff9d56

Reply:
It works with the latest Insiders. If no workspace is open, it still works.